core/pkg/monitor/monitor.go

```diff
@@ -130,7 +130,7 @@ func (sm *SystemMonitor) InitializeAssets(settings *spb.Settings) {
 	if network := NewNetwork(); network != nil {
 		sm.assets = append(sm.assets, network)
 	}
-	if gpu := NewGPUNvidia(sm.logger, pid, samplingInterval); gpu != nil {
+	if gpu := NewGPUNvidia(sm.logger, pid, samplingInterval, ""); gpu != nil {
```

Comment:
```suggestion
        cmdPath := ""
	if gpu := NewGPUNvidia(sm.logger, pid, samplingInterval, cmdPath); gpu != nil {
```
or alternatively options, i'm just thinking it would be more readable if we know what each argument represent?